api/features/permissions.py

```diff
@@ -185,3 +186,46 @@ def has_permission(self, request, view):
             permission=MANAGE_SEGMENT_OVERRIDES,
             environment=environment,
         )
+
+
+class FeatureImportPermissions(IsAuthenticated):
+    def has_permission(
+        self, request: Request, view: "WrappedAPIView"  # noqa: F821
+    ) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        environment = Environment.objects.get(id=view.kwargs["environment_id"])
+        return request.user.is_environment_admin(environment)
+
+
+class CreateFeatureExportPermissions(IsAuthenticated):
+    def has_permission(
+        self, request: Request, view: "WrappedAPIView"  # noqa: F821
+    ) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        environment = Environment.objects.get(id=request.data["environment_id"])
+        return request.user.is_environment_admin(environment)
+
+
+class DownloadFeatureExportPermissions(IsAuthenticated):
+    def has_permission(
+        self, request: Request, view: "WrappedAPIView"  # noqa: F821
+    ) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        feature_export = FeatureExport.objects.get(id=view.kwargs["feature_export_id"])
+
+        return request.user.is_environment_admin(feature_export.environment)
+
+
+class FeatureExportListPermissions(IsAuthenticated):
+    def has_permission(self, request: Request, view: ListAPIView) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        project = Project.objects.get(id=view.kwargs["project_id"])
+        return request.user.has_project_permission(VIEW_PROJECT, project)
```

Comment:
I feel like perhaps we should document why this is `VIEW_PROJECT` while the others require environment admin.

Reply:
Sounds good, I've added this.